dev/generate_schemas.py

```diff
@@ -1,240 +0,0 @@
-import json
```

Comment:
@chrisseto, I think we want this file. Can you please undelete it and move it to /scripts?